src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationItemFormatter.cs

```diff
@@ -2,72 +2,40 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Xml;
+using System.Runtime.Serialization;
+using System.Globalization;
+
 namespace System.ServiceModel.Syndication
 {
-    using System;
-    using System.Collections.Generic;
-    using System.Xml;
-    using System.Runtime.Serialization;
-    using System.Globalization;
-    using System.Xml.Serialization;
-    using System.Xml.Schema;
-    using System.Diagnostics.CodeAnalysis;
-    using DiagnosticUtility = System.ServiceModel.DiagnosticUtility;
-    using System.Runtime.CompilerServices;
-
     [DataContract]
     public abstract class SyndicationItemFormatter
     {
-        private SyndicationItem _item;
-
-        protected SyndicationItemFormatter()
-        {
-            _item = null;
-        }
+        protected SyndicationItemFormatter() => Item = null;
 
```

Comment:
I don't like converting constructors to delegate-based methods as it's non-obvious that it's been done. You were also inconsistent as the other constructor is also a one-liner.